stable/vpa/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: vpa
 description: A Helm chart for Kubernetes Vertical Pod Autoscaler
 type: application
-version: 1.4.0
-appVersion: 0.10.0
+version: 1.4.1
```

Comment:
Since this is a minor version change in the underlying project (VPA), I would prefer to make this a minor version change here as well.

Reply:
I would say that such a change in ["0 sem versioning"](https://0ver.org/) is a major change, but I don't want to trigger a major upgrade with no real breaking change. Anyway, I'm fine with using a minor version here!

I update my commit to release the `1.5.0` instead

Reply:
Great, thanks!